### PR TITLE
Refactor tests Part 1

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
## Description

Provide a summary of the modification as well as the issue that has been resolved.

- Limits the CLI tests to only one model to tests e2e functionality.
- Removes unnecessary methods from `conftest`.
## TODO

- [ ] Change task type depending on the model 

<details>
<summary>Describe the changes you made</summary>

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which refactors the code base)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

</details>

## Checklist

<details>
<summary>Ensure that you followed the following</summary>

- [ ] I have added a summary of my changes to the [CHANGELOG](https://github.com/openvinotoolkit/anomalib/blob/main/CHANGELOG.md) (not for minor changes, docs and tests).
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas (If applicable)
- [ ] I have made corresponding changes to the documentation (If applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works (If applicable)

</details>

### CODE REVIEW

File: tests/integration/cli/test_cli.py [hunk 1]
```diff
@@ -8,44 +11,32 @@
...
-            category = "UCSDped2"
-
         # batch size of 8 is taken so that the lr computation for efficient_ad does not return 0 when max_epochs is 1
         AnomalibCLI(
             args=[
                 "fit",
-                "-c",
-                f"src/configs/model/{model_name}.yaml",
+                "--model",
+                "Padim",
```
Comment (samet-akcay) [inline tests/integration/cli/test_cli.py:33]: are we gonna always use `Padim` or randomly choose it in the future?

**Review by samet-akcay — COMMENTED** [review]
Thanks @ashwinvaidya17, just a single comment/question only

File: tests/integration/cli/test_cli.py (hunk 1 above)
Comment (ashwinvaidya17) [reply tests/integration/cli/test_cli.py:33]: Yeah let's do that. The only issue is that AIVad does not work with MVTec but we can select from the rest.

Comment (samet-akcay) [reply tests/integration/cli/test_cli.py]: we could split image and video datasets/models in the future maybe?

**Review by samet-akcay — APPROVED** [review]

**Pull request merged**